api-specificatie/zrc/openapi.yaml

```diff
@@ -72,83 +75,95 @@ paths:
             application/json:
               schema:
                 $ref: '#/components/schemas/Fout'
+    parameters:
+      - name: version
+        in: path
+        required: true
+        schema:
+          type: string
+      - name: zaakid
+        in: path
+        required: true
+        description: zaakidentificatie
+        schema:
+          type: string
+servers:
+  - url: /
 components:
   schemas:
     Zaak:
-      description: Een zaak
+      description: Een ZAAK
+      required:
+        - zaaktype
+      type: object
       properties:
-        type:
-          description: URL naar het zaaktype van de zaak
-          type: string
-        omschrijving:
-          description: Omschrijving of toelichting op de zaak
-          type: string
-        initiator:
-          $ref: '#/components/schemas/Subject'
+        url:
+          title: Url
+          type: string
+          format: uri
+          readOnly: true
+        zaakidentificatie:
+          title: Zaakidentificatie
+          description: >-
+            De unieke identificatie van de ZAAK binnen de organisatie die
+            verantwoordelijk is voor de behandeling van de ZAAK.
+          type: string
+          maxLength: 40
+        zaaktype:
+          title: Zaaktype
+          description: URL naar het zaaktype in de CATALOGUS waar deze voorkomt
+          type: string
+          format: uri
+          maxLength: 200
+          minLength: 1
         status:
-          description: URL naar de status van de zaak
+          title: Status
+          description: 'Indien geen status bekend is, dan is de waarde ''null'''
           type: string
+          format: uri
+          readOnly: true
```

Comment:
Als dit readonly is, hoe kan je dan een zaak update doen?

Reply:
Dit loopt wat voor op https://github.com/maykinmedia/gemma-zaakregistratiecomponent/blob/54e8f6c7d9168895a55ea89c0163c81e1f782fa9/src/openapi.yaml#L14

Het idee is dat je straks een nieuwe status POST om een `Zaakstatus` aan te maken/te zetten. Bij het uitlezen van de zaak kan je altijd de dan-geldende status terugkrijgen (als referentie)